command/results_xunit_parser_test.go

```diff
@@ -205,6 +196,10 @@ func TestXMLToModelConversion(t *testing.T) {
 					So(len(logs[0].Lines), ShouldNotEqual, 0)
 					So(logs[0].URL(), ShouldContainSubstring,
 						"TEST/5/test.test_auth.TestAuthURIOptions.test_uri_options")
+					Convey("and system-out and system-err should be present", func() {
+						So(logs[len(logs)-1].Lines[0], ShouldContainSubstring, "system-err: system error text")
+						So(logs[len(logs)-2].Lines[0], ShouldContainSubstring, "system-out: system out text")
+					})
```

Comment:
can we add some additional tests of cases with one or neither?

Reply:
Added some more tests